npc/quests/quests_ein.txt

```diff
@@ -474,18 +474,487 @@ einbech,97,167,5	script	Cavitar	4_M_EINOLD,{
+				mes "sugar honey! Oh, you";
+				mes "cutie adventurers are";
+				mes "so precious, so lovable.";
+				mes "^333333*Tee hee hee~*^000000";
+				next;
+				mes "[Uwe]";
+				mes "Well, that was";
+				mes "a fun talk. Come";
+				mes "back and visit, 'kay?";
+				mes "I'll miss you until";
+				mes "the next time~";
+				close;
+			}
+			mes "[Uwe]";
+			mes "Big cities can be nice,";
+			mes "but you never get to enjoy";
+			mes "the sensation of stepping";
+			mes "barefoot through the grass,";
+			mes "or the magnificent sight of";
+			mes "the shining stars at night.";
+			next;
+			mes "[Uwe]";
+			mes "And the people who live";
+			mes "in the cities can be so";
+			mes "uptight! I've known more";
+			mes "than a few who look down";
+			mes "on people from smaller towns.";
+			next;
+			mes "[Uwe]";
+			mes "It's sad to see some";
+			mes "people will always be";
+			mes "that ignorant. I... I just";
+			mes "can't believe those people";
+			mes "can be sooo close minded!";
+			next;
+			mes "[Uwe]";
+			mes "Oh...!";
+			mes "I'm sorry, I guess";
+			mes "I went a little overboard.";
+			mes "^666666*Titter*^000000 Next time you drop";
+			mes "by, we'll talk about something";
+			mes "more fun, 'kay? Buhbye~";
+			close;
+		}
+	}
+	
+	function F_ForgeWeapon {
+		mes "[Uwe]";
+		mes "Well...";
+		mes "I actually just";
+		mes "do smithing work";
+		mes "to create my own";
+		mes "cooking tools.";
+		next;
+		mes "[Uwe]";
+		mes "Oh, I understand";
+		mes "that somebody needs";
+		mes "to fight the monsters,";
+		mes "but I'm the wrong person";
+		mes "to ask for forging weapons.";
+		mes "I... am a strict pacifist~";
+		next;
+		mes "[Uwe]";
+		mes "Just go look";
+		mes "around for a little";
+		mes "bit, I'm sure you'll";
+		mes "find a Blacksmith";
+		mes "who's willing to forge";
+		mes "you a good weapon~";
+		close;
+	}
+	
+	function F_Cancel {
+		mes "[Uwe]";
+		mes "Oh...?";
+		mes "Well, feel";
+		mes "free to visit";
+		mes "me whenever";
+		mes "you want, 'kay?";
+		close;	
+	}
+	
+	function F_RandCloseMsg {
+		switch (rand(1, 3)) {
+		case 1:
+			mes "...Ho ho~";
+			close;
+		case 2:
+			mes "...*Tee Hee~*";
+			close;
+		case 3:
+			mes "...Behbie~";
+			close;
+		}
+	}
+	
+	function F_Ingredients {
+		mes "[Uwe]";
+		mes "Now for beginners,";
+		mes "learning to select";
+		mes "and use ingredients";
+		mes "is one of the most";
+		mes "important fundamentals.";
+		next;
+		mes "[Uwe]";
+		mes "Cooking is like forging";
+		mes "since specific ingredients";
+		mes "are needed to make specialty";
+		mes "items or dishes. You can't just";
+		mes "skip them if you really need";
+		mes "them, right? Right!";
+		next;
+		mes "[Uwe]";
+		mes "Once you make up your";
+		mes "mind to do something,";
+		mes "focus on finishing it, 'kay?";
+		mes "Never cut corners and always";
+		mes "dedicate yourself to make the";
+		mes "very best finished product~";
+		if (ein_cook == 7 && BaseClass == Job_Blacksmith)
```

Comment:
I don't think this is according to proper Hercules code styling.
Is there a parser-bug why'd you recommend me to do that?